app/models.py

```diff
@@ -140,6 +140,17 @@ class User(db.Model):
     def password(self):
         raise AttributeError("Password not readable")
 
+    @property
+    def can_use_webauthn(self):
+        if self.platform_admin:
+            return True
+
+        return any([
+            str(service.organisation_id) == current_app.config['BROADCAST_ORGANISATION_ID'] or
+            str(service.id) == current_app.config['NOTIFY_SERVICE_ID']
+            for service in self.services
+        ])
```

Comment:
You can remove the square brackets form this, that way it'll be passed in as a generator rather than evaluated in a list comprehension, which will speed it up as `any` can shortcut return as soon as the first matching service is seen.

Reply:
Nice, that's a bit shorter! 

I'd forgotten about generators, and also that you don't need extra parentheses for them inside a function call.